Tests/Runtime/Random/RandomStringTest.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2023 Koji Hasegawa.
+// This software is released under the MIT License.
+
+using System.Linq;
+using NUnit.Framework;
+
+namespace TestHelper.Monkey.Random
+{
+    [TestFixture]
+    public class RandomStringTest
+    {
+        [Test]
+        public void NextDigits()
+        {
+            // Property-based testing
+            for (var i = 0; i < 1000; i++)
```

Comment:
Use RepeatAttribute.
(However, it can only be used with sync test)

Reply:
Ah, I didn't know the attribute. I'm gonna use it.

Reply:
Hmm, I thought the Repeat attribute can give a repeat count, but seems not be.

I intended that the test be deterministic, but using the attribute make the test non-deterministic. I think deterministic is better than non-deterministic here.

Reply:
Try
`[Repeat(1000)]`

maybe: you're confusing it with RetryAttribute?

Reply:
Sorry, I misunderstood.
Keep it, please.

Maybe can get count from test context. However, it is a hassle so leave it as is.

Reply:
> maybe: you're confusing it with RetryAttribute?

No. I intended loop-counter, not "repeat count". I'm sorry about it.

Loop-counter is used to make test be deterministic and to choose a number from wide-range: